app/assets/stylesheets/application.scss

```
@@ -32,3 +32,17 @@
 .alert-alert {
   @extend .alert-danger;
 }
+
+// ナビバーの配置調整
+header {
+  margin-bottom: 100px;
+}
+
+ul {
```

影響範囲が大きくなりすぎますので，せめて `.navbar` あたりにした方がよいでしょう。

```suggestion
.navbar {
```